src/main/java/net/daporkchop/fp2/asm/core/server/IMixinWorldBorder.java

```diff
@@ -0,0 +1,14 @@
+package net.daporkchop.fp2.asm.core.server;
+
+import net.minecraft.world.border.WorldBorder;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+@Mixin(WorldBorder.class)
+public interface IMixinWorldBorder {
+    @Accessor("startDiameter")
+    double getWorldBorderStartDiameter();
```

Comment:
this is never used, can be removed (non-issue since you should replace the whole mixin with an access transformer)